meerk40t/core/elements/element_treeops.py

```diff
@@ -167,24 +167,49 @@ def image_convert_modifier(node, **kwargs):
             type="elem image",
         )
 
+    @tree_operation(_("Remove effect"), node_type=effect_nodes, help="")
+    def remove_effect(node, **kwargs):
+        childs = [e for e in node._children]
+        for e in childs:
+            e._parent = None  # Otherwise add_node will fail below
+            node.parent.add_node(e)
+        node._children.clear()
+        node.remove_node()
+        self.signal("rebuild_tree")
+
+
     @tree_conditional(lambda node: is_hatched(node))
```

Comment:
The is_hatched command is correct but confusingly named since it does `is_effected` now.